### DISCUSSION TITLE
MagickReadSettings vs. MagickImage

### DISCUSSION THREAD

**Discussion by wyeack** [dlemstra/Magick.NET#645] [Help]
In alot of the examples, I see people setting properties on MagickReadSettings and padding it into the MagickImage when it is being created.  However, all of the fields that I have found in MagickReadSettings are also in MagickImage - so when would I set the field on MagickReadSettings vs. set it on MagickImage?

**Comment by dlemstra** [comment]
The `MagickReadSettings` are use when you want to set some settings before you read the image. One example would be the `Density` when reading a vector (e.g. SVG/PDF) image. The settings on the image are when you want to change something after the image has been read.
